library/bignum_core.h

```diff
@@ -290,4 +290,23 @@ void mbedtls_mpi_core_montmul( mbedtls_mpi_uint *X,
                                const mbedtls_mpi_uint *N, size_t AN_limbs,
                                mbedtls_mpi_uint mm, mbedtls_mpi_uint *T );
 
+/**
+ * \brief Calculate the square of the Montgomery constant. (Needed
+ *        for conversion and operations in Montgomery form.)
+ *
+ * \param[out] X  A pointer to the result of the calculation of
+ *                the square of the Montgomery constant:
+ *                2^{2*n*biL} mod N.
+ * \param[in]  N  Little-endian presentation of the modulus, which must be odd.
+ *
+ * \return        0 if successful.
+ * \return        #MBEDTLS_ERR_MPI_ALLOC_FAILED if there is not enough space
+ *                to store the value of Montgomery constant squared.
+ * \return        #MBEDTLS_ERR_MPI_DIVISION_BY_ZERO if \p N modulus is zero.
+ * \return        #MBEDTLS_ERR_MPI_NEGATIVE_VALUE if \p N modulus is negative.
+ * \return        #MBEDTLS_ERR_ERROR_CORRUPTION_DETECTED if other operations fail.
+ */
+int mbedtls_mpi_core_get_mont_R2_unsafe( mbedtls_mpi *X,
+                                         mbedtls_mpi const *N );
```

Comment:
same comment (as opposed to a request for change) about the order of `mbedtls_mpi` and `const` - see (e.g.) line 290 above

Reply:
Changed as requested